src/config/queue.ts

```diff
@@ -84,7 +89,58 @@ async function initializePatientQueue(conn: Connection) {
                     `Message Queue: Received ${testPayload.length} test data`
                 );
                 if (testPayload.length > 0) {
-                    // TODO: Preprocess and load to database
+                    //TODO: Preprocess and load to database
+                    var newZipMap = new Map<Number, Number>();
+                    zipList = [];
+                    const now = new Date()
+                    //this is so scuffed
+                    //making a map of the new message
+                    for(let i = 0; i < testPayload.length; i++){
+                        if(Number(testPayload[i]["patient_status"]) == 1){
```

Comment:
I just noticed this but there is a chance that he will give us unclean data for `patient_status` which may break your part here. `patient_status` can be something like "Hello World" and will create an error here.